tests/integration.rs

```diff
@@ -1458,10 +1458,15 @@ fn _test_virtio_vsock(hotplug: bool) {
         }
 
         // Validate vsock works as expected.
-        guest.check_vsock(socket.as_str());
+        guest.check_vsock_host_to_guest(socket.as_str());
         guest.reboot_linux(0, None);
         // Validate vsock still works after a reboot.
-        guest.check_vsock(socket.as_str());
+        guest.check_vsock_host_to_guest(socket.as_str());
+
+        guest.check_vsock_guest_to_host(socket.to_string());
+        guest.reboot_linux(0, None);
```

Comment:
You don't need to reboot twice.

Instead, you can just add the guest_to_host test to the previous section.

Reply:
Indeed. Fixed that. Btw. this code doesn't seem to run, the fmethod starts with `_`.

Thanks

Reply:
Oh, actually this code is run, just the method is included in two different tests with different arguments passed.

Thanks